test/ussd_public.test.js

```diff
@@ -148,6 +148,73 @@ describe("app", function() {
             });
         });
 
+        describe("dialback sms testing", function() {
+            it("send if redial sms not yet sent", function() {
+                return tester
+                .setup.user.addr("27820001001")
+                .inputs(
+                    {session_event: 'new'}  // dial in
+                    , "1"  // state_language - isiZulu
+                    , {session_event: 'close'}
+                )
+                .check.user.answer("redial_sms_sent", true)
+                .check(function(api) {
+                    utils.check_fixtures_used(api, [125, 160, 163]);
+                })
+                .run();
+            });
+            it("don't send if redial sms already sent", function() {
+                return tester
+                .setup.user.addr("27820001011")
+                .inputs(
+                    {session_event: 'new'}  // dial in
+                    , "1"  // state_registered_full - compliment
+                    , {session_event: 'close'}
+                )
+                .check.user.answer("redial_sms_sent", true)
+                .check(function(api) {
+                    utils.check_fixtures_used(api, [53, 54, 127, 187]);
+                })
+                .run();
+            });
+            it("updates identity with redial_sms_sent 'false'", function() {
```

Comment:
Change this description to `should not send dial-back sms when timeout occurs on a non-dialback state` or something similar.